src/devices/GrovePi/GrovePiDevice.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <RootNamespace>Iot.Device.GroovePiDevice</RootNamespace>
+    <LangVersion>7.3</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease.19078.2" />
```

Comment:
This is not right. Can you please instead use version $(SystemDeviceGpioPackageVersion) ? That is what we are doing in the rest of the repo

Reply:
will adjust, the PR is so old that it was not the case originally. Thanks for checking! Will adjust as well